modules/bim/lib/open_project/bim/patches/attachment_patch.rb

```diff
@@ -44,8 +44,8 @@ def content_disposition(include_filename: true)
       ifc_content_disposition
     end
 
-    def ifc_model?
-      container_type == Bim::IfcModels::IfcModel.name && container.present?
+    def ifc_file?
+      container_type == Bim::IfcModels::IfcModel.name && extension == ".ifc" && container.present?
```

Comment:
@machisuji it might be even safer to check the `description` to be "ifc" instead of `extension`. The description in the case of `IfcModel` attachments contains what the file is intended to be. It could also be `xkt`. The extension contains what the suffix of a file was when it was uploaded, right?

Reply:
Yes good idea, I did that and fixed the check for `ifc_model?` which is not a method